content/blog/2020-07-24-first-mlops-tutorial.md

```diff
@@ -0,0 +1,67 @@
+---
+title: |
+  NEW VIDEO! 🎥 MLOps Tutorial #1: 
+  Intro to continuous integration for ML
+date: 2020-07-24
+description: |
+  A video tutorial abut using continuous integration in data 
+  science and machine learning projects. This tutorial shows 
+  how to use GitHub Actions and Continuous Machine Learning 
+  (CML) to create your own automated model training and 
+  evaluation system.
+picture: 2020-07-24/blog_header.png
```

Comment:
please compress/optimize the image trough some online tool

Reply:
just did and pushed the change